Editor/Resources/EditorWindow/Components/DeploymentScenarios.uxml

```diff
@@ -0,0 +1,41 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <Style src="../common.uss"/>
+    <ui:GroupBox class="separator">
+        <ui:VisualElement class="form-row form-row--free">
+            <ui:Label name="ManagedEC2ScenarioSingleFleetLabel" class="form-row__label" text="Single-Region Fleet"/>
+            <ui:VisualElement class="form-row__input">
+                <ui:RadioButton name="ManagedEC2ScenarioSingleFleetRadio" class="form-row__radio-button" text="Creates a game backend service with a single Amazon GameLift fleet."/>
+                <ui:VisualElement class="external-link">
+                    <ui:Label name="ManagedEC2ScenarioSingleFleetLink" class="form-row__text-display input-hint foldout__link" text="Learn More"/>
+                    <ui:Image name="ExternalLink" class="icon"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement name="FleetTypeRadioButton2Group" class="form-row form-row--free">
+            <ui:Label name="ManagedEC2ScenarioSpotFleetLabel" class="form-row__label" text="Spot Fleet"/>
+            <ui:VisualElement class="form-row__input">
+                <ui:RadioButton name="ManagedEC2ScenarioSpotFleetRadio" class="form-row__radio-button" text="Forms matches by using Amazon GameLift queues and a custom matchmaker and configures three fleets."/>
+                <ui:VisualElement class="external-link">
+                    <ui:Label name="ManagedEC2ScenarioSpotFleetLink" class="form-row__text-display input-hint foldout__link" text="Learn More"/>
+                    <ui:Image name="ExternalLink" class="icon"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:VisualElement>
+        <ui:VisualElement name="FleetTypeRadioButton3Group" class="form-row form-row--free">
+            <ui:Label name="ManagedEC2ScenarioFlexFleetLabel" class="form-row__label" text="Flex Match"/>
+            <ui:VisualElement class="form-row__input">
+                <ui:RadioButton name="ManagedEC2ScenarioFlexFleetRadio" class="form-row__radio-button" text="Uses FlexMatch, a managed matchmaking service, to match game players together."/>
```

Comment:
Consider feedback about naming conventions. For some context the two types of FlexMatch are `Standalone` and `WithQueue`. The latter refers to a Configuration being connected to Fleets. I would consider renaming this to `FlexMatchWithQueue`

https://docs.aws.amazon.com/gamelift/latest/apireference/API_CreateMatchmakingConfiguration.html#gamelift-CreateMatchmakingConfiguration-request-FlexMatchMode